src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -120,7 +206,6 @@
     }
 
     private boolean isSolicitorAgreedToReceiveEmails(Map<String, Object> mapOfCaseData, String solicitorAgreeToReceiveEmails) {
```

Comment:
there are also other classes with similar methods, such as:

```
    private boolean applicantIsNotRepresentedByASolicitor(Map<String, Object> caseData) {

        return NO_VALUE.equalsIgnoreCase(nullToEmpty(caseData.get(APPLICANT_REPRESENTED)));
    }

    private boolean solicitorDidNotAgreeToReceiveEmails(Map<String, Object> caseData) {

        return NO_VALUE.equalsIgnoreCase(nullToEmpty(caseData.get(SOLICITOR_AGREE_TO_RECEIVE_EMAILS)));
    }
```

Why don't we move them to `CommonConditions` class and unit test it easily :)